akondas/symfony-consul-bundle/0.1/config/packages/consul.yaml

```diff
@@ -0,0 +1,7 @@
+consul:
+    service:
+        name: 'symfony-app'
+        host: 'localhost'
+        port: 8000
+    client:
+        base_uri: http://127.0.0.1:8500
```

Comment:
Please correct me if Im wrong. Doesnt all these values change depending on if you are on local development or in production?

If so, they should be environment variables.

Reply:
it depends on the approach, but the idea to give environment variables there is absolutely justified

Reply:
Could you please update the PR accordingly?

Reply:
Updated, I'm sorry that it took so long.